lib/install/elm.rb

```diff
@@ -12,9 +15,9 @@
           "#{Webpacker::Configuration.entry_path}/hello_elm.js"
 
 puts "Installing all elm dependencies"
-run "yarn add elm elm-webpack-loader"
-run "yarn add --dev elm-hot-loader"
-run "yarn run elm package install -- --yes"
+run "#{node_bundler} elm elm-webpack-loader"
+run "#{node_bundler} --dev elm-hot-loader"
+run "#{node_bundler} elm package install -- --yes"
```

Comment:
i think this might need to be `${Webpacker::Configuration.node_modules_bin_path}/elm`?